packages/grafana-eslint-rules/README.md

```diff
@@ -12,6 +12,12 @@ Previously we hijacked the aria-label property to use as E2E selectors as an att
 
 Now, we prefer using data-testid for E2E selectors.
 
+### `@grafana/no-border-radius-literal`
+
+Check if border-radius theme tokens are used.
+
+To improve the consistency across Grafana we encourage devs to use tokens instead of custom values. In this case, we want the `borderRadius` to use the appropiate token such as `theme.shape.borderRadius()` or `theme.shape.radius.circle`.
```

Comment:
Is `theme.shape.borderRadius()` an appropriate token? Isn't it everything under `theme.shape.radius`?

Reply:
Yes, we should avoid `theme.shape.borderRadius()` as it's deprecated. I modify this line and now the info is updated :)